app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -1107,21 +1106,28 @@ public void draw(Canvas aCanvas) {
         }
     }
 
-    public void setSession(@NonNull Session aSession) {
-        setSession(aSession, SESSION_RELEASE_DISPLAY);
+    public void setSession(@NonNull Session aSession, boolean aDeactivateCurrentSession) {
+        setSession(aSession, SESSION_RELEASE_DISPLAY, aDeactivateCurrentSession);
     }
 
-    public void setSession(@NonNull Session aSession, @OldSessionDisplayAction int aDisplayAction) {
+    public void setSession(@NonNull Session aSession, @OldSessionDisplayAction int aDisplayAction, boolean aDeactivateCurrentSession) {
```

Comment:
I would prefer something other than a boolean be used for `aDeactivateCurrentSession` since it makes the call sites harder to read.

Reply:
Reverted to use an IntDef